.env

```diff
@@ -1,3 +1,5 @@
 PORT=3001
+REACT_APP_PREVIEW_SERVER=http://localhost:8000
 REACT_APP_QUEUE_API_PREFIX=qs
+REACT_APP_PREVIEW_API_PREFIX=preview
 REACT_APP_FACILITY_API_PREFIX=info
```

Comment:
Let's sort them alphabetically, so it's easier to find things as this list grows.